common/scripts/generate-doc.js

```diff
@@ -36,11 +36,64 @@ function walk(dir, checks) {
   return checks;
 }
 
+function buildList(inputString) {
+  var inputList = inputString[1].split(",");
```

Comment:
why inputstring[1]? What's in inputstring[0]?

Reply:
Removed this function in favour of arguments handler package for js => `yargs`

Reply:
Nice! Resolve these as you eliminate them.